openpype/hosts/maya/plugins/publish/validate_resolution.py

```diff
@@ -0,0 +1,100 @@
+import pyblish.api
+from openpype.pipeline import (
+    PublishValidationError,
+    OptionalPyblishPluginMixin
+)
+from maya import cmds
+from openpype.pipeline.publish import RepairAction
+from openpype.hosts.maya.api import lib
+from openpype.hosts.maya.api.lib import reset_scene_resolution
+
+
+class ValidateResolution(pyblish.api.InstancePlugin,
+                         OptionalPyblishPluginMixin):
+    """Validate the render resolution setting aligned with DB"""
+
+    order = pyblish.api.ValidatorOrder - 0.01
```

Comment:
Out of curiosity, is there are reason we check this a validator order - 0.01? Why not just `ValidatorOrder`?

Reply:
Great - let's remove it.
```suggestion
    order = pyblish.api.ValidatorOrder
```